### ISSUE TITLE
CED Viewer crashes whith input

### ISSUE THREAD

**Issue by GarcaHec** [iLCSoft/CEDViewer#12]
I have been following this tutorial:

    https://agenda.linearcollider.org/event/7520/contributions/38806/attachments/31480/47409/gaede_ilcsoft_tutorial.pdf 

When I come to the point of using glced or ced2go (slide 22) to visualize events, the commands run without error and I get the expected window with the detector geometry and the events displayed. However, the moment I rotate the 3D image or I hover the mouse over the menu the application crashes.

- OS version: CentOS Linux release 7.7.1908 (Core)
- Compiler version: gcc (GCC) 4.9.3
- Package version: /cvmfs/ilc.desy.de/sw/x86_64_gcc49_sl6/v02-00-01
- Reproduced by: ced2go -d /cvmfs/ilc.desy.de/sw/ILDConfig/v02-00-01/StandardConfig/production/Gear/gear_ILD_l5_v02.xml pi_80GeV_1000evt_REC.slcio
- Input: Reconstructed file -> wwwae.ciemat.es/~carrillo/calice/datasets/pi_80GeV_1000evt_REC.slcio
- Output: [log.txt](https://github.com/iLCSoft/CEDViewer/files/4450505/log.txt)
- Goal: Display events for analysis.

**Comment by rete** [comment]
Hi,
thanks for reporting. 
Your error looks like a server / client issue. OpenGL is capricious in terms of client / server OS compatibility, in particular when it comes to which side renders the scene. 

Are you running the event display through ssh ? 
If yes, what is the client OS ? 

I guess the server OS where the software is installed is the one you mentioned in the issue header (Centos 7).

**Comment by GarcaHec** [comment]
I am running the display locally in my computer (no ssh and with the OS mentioned) using ced2go from the /cvmfs/ installation.

**Comment by rete** [comment]
So you have installed CVMFS on your machine and mounted the `ilc.desy.de` endpoint ?

If so, I'm not sure your Centos7 will be compatible with our SL6 deployment on CVMFS, in particular when it comes to OpenGL drivers from CVMFS versus from your machine.

@andresailer @petricm Any idea ?

**Comment by GarcaHec** [comment]
Yes. In particular I have the Centos7 provided by CERN here: http://linux.web.cern.ch/linux/centos7/
